WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/Edit Order Status/OrderStatusListViewController.swift

```diff
@@ -216,11 +221,11 @@ private extension OrderStatusListViewController {
 //
 extension OrderStatusListViewController: UITableViewDataSource {
     func numberOfSections(in tableView: UITableView) -> Int {
-        return statusResultsController.sections.count
+        return 1
```

Comment:
If you do not implement this method, the table configures the table with one section. 👍